pom.xml

```diff
@@ -25,7 +25,7 @@
   </licenses>
 
   <properties>
-    <revision>1.27.6</revision>
+    <revision>1.27.5.2</revision>
```

Comment:
Is this really supposed to be here @daniel-beck @olamy?

Reply:
Good catch nope
I will fix that

Reply:
fixed with 15a25be31